src/main/kotlin/io/github/mojira/arisa/ModuleRegistry.kt

```diff
@@ -203,22 +208,24 @@ class ModuleRegistry(jiraClient: JiraClient, private val config: Config) {
             )
         }
 
-        // TODO FIX THIS SOMEHOW TO REUSE THE FUNCTIONS FROM TRANSFERVERSION - NEED HELP
-        /* register(
+        register(
              "TransferLinks",
              Modules.TransferLinks,
              TransferLinksModule()
-         ) { issue ->
-             val links =
-                 issue.getLinks(jiraClient, ::createLinkForTransfer, ::getIssueForLink.partially1(jiraClient))
+        ) { issue ->
+            val links = issue.getLinks<List<Link<*, LinkParam>>, LinkParam>(
+                    jiraClient,
+                    ::createLinkForTransfer,
+                    ::getIssueForLink.partially1(jiraClient)
+                )
 
 
-             AbstractTransferFieldModule.Request(
-                 issue.key,
-                 links,
-                 links
-             )
-         }*/
+            AbstractTransferFieldModule.Request(
+                issue.key,
+                links,
+                links
```

Comment:
If you have a better idea for that module without reusing too much code between this and TransferVersions, feel free btw 😉